app/templates/partials/footer.html

```diff
@@ -108,6 +121,8 @@
            renderChatbox();
            a11y();
            checkReload();
+           CalculatePageLoadTime();
```

Comment:
Does this need to get called on every line change? And can `ldTrackLoadTime` get added to this function instead of a separate one? Is there a need for both?

Reply:
Done, check new PR. I moved everything into 1 function and track gets called there too.